rest_framework_social_oauth2/oauth2_grants.py

```diff
@@ -90,6 +91,11 @@ def validate_token_request(self, request):
 
         try:
             user = backend.do_auth(access_token=request.token)
+            user_data = backend.user_data(access_token=request.token)
+            exp = user_data['exp']
```

Comment:
what's the value exp holds?

Reply:
`exp` is the expiration date of the token